src/frontend/Navigation/Stack/DeviceNamingScreens.tsx

```diff
@@ -1,16 +1,33 @@
 import * as React from 'react';
 import {RootStack} from '.';
-import {IntroToCoMapeo} from '../../screens/IntroToCoMapeo';
-import {DeviceNaming} from '../../screens/DeviceNaming';
-import {Success} from '../../screens/Success';
+import {IntroToCoMapeo} from '../../screens/Onboarding/IntroToCoMapeo';
+import {DataPrivacy} from '../../screens/Onboarding/DataPrivacy';
+import {OnboardingPrivacyPolicy} from '../../screens/Onboarding/OnboardingPrivacyPolicy';
+import {DeviceNaming} from '../../screens/Onboarding/DeviceNaming';
+import {Success} from '../../screens/Onboarding/Success';
+import {MessageDescriptor} from 'react-intl';
 
-export const createDeviceNamingScreens = () => (
+export const createDeviceNamingScreens = ({
```

Comment:
nit: related to just naming stuff, but this should probably be using `onboarding` instead of `deviceNaming`. same for other places that are using `deviceNaming` naming